packages/components/modules/activity-log/DateFilterComponent/types.ts

```diff
@@ -0,0 +1,17 @@
+import { DateTime } from 'luxon'
+
+export interface IFetchParameters {
```

Comment:
could you move this to `packages/components/modules/activity-log/common/types.ts`? it's probably going to be useful for native implementation, and it's something that is being reused as a log, so we should put it in a more accessible place.

Also rename it to `IFetchParameters`, we're trying to get rid of the `I` prefix 🙏🏽